tchannel/testing/vcr/exceptions.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+
+
+class VCRException(Exception):
```

Comment:
@breerly Grayson suggested we call 'Error' instead of 'Exception' long time ago. The base VCRException should also be changed to VCRError